src/main/java/uk/gov/hmcts/darts/arm/service/impl/ArmBatchProcessResponseFilesImpl.java

```diff
@@ -557,40 +592,73 @@ private void readInvalidLineFile(BinaryData invalidLineFileBinary, InvalidLineFi
+
+                        // Read the invalid lines file and log the error code and description with EOD
+                        log.warn(
+                            "ARM invalid line for external object id {}. ARM error description: {} ARM error status: {}",
+                            externalObjectDirectory.getId(),
+                            armResponseInvalidLineRecord.getExceptionDescription(),
+                            armResponseInvalidLineRecord.getErrorStatus()
+                        );
+                        updateTransferAttempts(externalObjectDirectory);
+                        externalObjectDirectory.setErrorCode(armResponseInvalidLineRecord.getExceptionDescription());
+                        updateExternalObjectDirectoryStatus(externalObjectDirectory, EodHelper.armResponseManifestFailedStatus());
+                    } else {
+                        String error = String.format("Incorrect status [%s] for invalid line file %s", invalidLineFileFilenameProcessor.getStatus(),
+                                                     invalidLineFileFilenameProcessor.getInvalidLineFileFilenameAndPath());
+                        log.warn(error);
+                        externalObjectDirectory.setErrorCode(error);
+                        updateExternalObjectDirectoryStatus(externalObjectDirectory, EodHelper.armResponseProcessingFailedStatus());
+                    }
                 } else {
-                    String error = String.format("Incorrect status [%s] for invalid line file %s", invalidLineFileFilenameProcessor.getStatus(),
-                                  invalidLineFileFilenameProcessor.getInvalidLineFileFilenameAndPath());
-                    log.warn(error);
-                    externalObjectDirectory.setErrorCode(error);
+                    log.warn("Unable to read invalid line file {}", invalidLineFileFilenameProcessor.getInvalidLineFileFilenameAndPath());
                     updateExternalObjectDirectoryStatus(externalObjectDirectory, EodHelper.armResponseProcessingFailedStatus());
                 }
             } else {
-                log.warn("Unable to read invalid line file {}", invalidLineFileFilenameProcessor.getInvalidLineFileFilenameAndPath());
-                updateExternalObjectDirectoryStatus(externalObjectDirectory, EodHelper.armResponseProcessingFailedStatus());
+
+                String armResponseFile = getOtherFailedArmResponseFile(createRecordFilenameProcessor, uploadFileFilenameProcessor);
+
+                log.warn("Unable to find external object directory with ID {} for ARM batch responses with IL file {}, other response file{}",
+                         externalObjectDirectoryId,
+                         invalidLineFileFilenameProcessor.getInvalidLineFileFilenameAndPath(),
+                         armResponseFile);
+                List<String> invalidResponseFiles = getInvalidResponseFiles(invalidLineFileFilenameProcessor, armResponseFile);
```

Comment:
this only deletes the IL file but what about if IL also has a CR or UF file associated? Are we deleting them somewhere else?